ui/package.json

```diff
@@ -33,7 +33,7 @@
     "test:ju:report": "junit-viewer --results=junit-results --save-file=cypress/site/junit-report.html",
     "test:ju:clean": "rm junit-results/*.xml",
     "test:ju:all": "yarn test:ju:clean && yarn test:junit; yarn test:ju:report;",
-    "test:ci": "JEST_JUNIT_OUTPUT_DIR=\"./coverage\" jest --ci --coverage --reporters=default --reporters=jest-junit",
+    "test:ci": "JEST_JUNIT_OUTPUT_DIR=\"./coverage\" jest --ci --coverage --maxWorkers=2",
```

Comment:
this run command was added specifically for making jenkins tests run good within the CI pipeline. can you make sure that's still the case with these changes?

Reply:
Done!